libcodechecker/cmd/cmd_line_client.py

```diff
@@ -332,6 +397,24 @@ def printReports(client, reports, output_format):
             print(CmdLineOutputEncoder().encode(output))
             return
 
+        if output_format == 'html':
+            if len(reports) == 0:
+                print('No {0} reports was found!'.format(diff_type))
+                return
+
+            output_dir = args.export_dir
+            if not os.path.exists(output_dir):
+                os.makedirs(output_dir)
+
+            print("Generating HTML output files to {0} directory:\n".format(
+                output_dir))
+
+            report_to_html(client, reports, output_dir)
+
+            print('\nTo view the results in a browser run:\n'
+                  '> firefox {0}'.format(args.export_dir))
```

Comment:
What is `>`? Linux prompt character is `$`.

Reply:
If you print the directory path in this format `file://home/myuser/report_directory` it will be clickable in the terminal right?

Reply:
In most Desktop terminal emulators, such as `gnome-terminal`, yes. I think the ancient `xterm` isn't capable of such feat.